.gitignore

```diff
@@ -28,27 +35,17 @@ manifest.yml
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
-
-# Vim
-# Persistent undo
-[._]*.un~
-# Temporary
-.netrwhist
-*~
```

Comment:
Please keep the VIM related files to ignore.

Reply:
Fixed